php-packages/phpstan/stubs/Illuminate/Contracts/Filesystem/Cloud.stub

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Illuminate\Contracts\Filesystem;
+
+interface Cloud extends Filesystem
```

Comment:
We'll want to add phpdoc to the filesystem driver such that it only allows class strings implementing Cloud

Reply:
Sorry I don't follow 🤔

Reply:
I meant we should change the extender's phpdocs so that they use `class-string` of an interface annotations instead of just string. If/when we roll out PHPStan to extensions, that should ensure that they are being used properly.